tutorials/quickstart/tutorial.md

```diff
@@ -22,18 +22,18 @@ You'll also see how the additional tools in the Particular Service Platform make
 
 #variant Variant
 
-In this tutorial, you'll see how NServiceBus takes the grunt work out of system design by handling all of the plumbing for you.
+In this tutorial, you'll see how NServiceBus takes the grunt work out of system design by handling all the plumbing for you.
 
-You will explore part of an e-commerce system that handles receiving an order placed by a user, as well as the billing and shipping that goes along with it. This is what the final solution will look like:
+You'll explore part of an e-commerce system that handles receiving an order placed by a user, and the billing and shipping that goes with it. The final solution will look like this:
 
 ![quickstart solution diagram](/tutorials/quickstart/after.svg)
 
-The tutorial is composed of 3 parts - introduction, failure recovery, and extensibility - that demonstrate how NServiceBus helps with design best practices and how the Particular Service Platform helps monitor a distributed system. Each part takes approximately 15-20 minutes to complete.
+The tutorial has 3 parts—an introduction, failure recovery, and extensibility—that demonstrate how NServiceBus helps with best design practices and how the Particular Service Platform helps monitor a distributed system. Each part takes about 15-20 minutes to complete.
```

Comment:
> ...how NServiceBus helps with design best practices...

It sounds the activity is "designing best practices", and the sentence has a grammar mistake.

> ...how NServiceBus helps with best design practices...

This doesn't sound too great either, so maybe just:

```suggestion
The tutorial has 3 parts—an introduction, failure recovery, and extensibility—that demonstrate how NServiceBus helps with best practices and how the Particular Service Platform helps monitor a distributed system. Each part takes about 15-20 minutes to complete.
```